.github/dependabot.yml

```diff
@@ -26,4 +26,22 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types:
-          ["version-update:semver-major", "version-update:semver-patch"]
+          ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
+  - package-ecosystem: pip
+    directory: "/cirrus/server"
+    schedule:
+      interval: weekly
+    target-branch: main
+    ignore:
+      - dependency-name: "*"
+        update-types:
+          ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
+  - package-ecosystem: pip
```

Comment:
I'll have to add `npm` to this config in the PR that adds npm publishing to schemas.

Reply:
Yep 👍